packages/lib/onedrive-api.js

```diff
@@ -4,6 +4,8 @@ const time = require('./time').default;
 const Logger = require('./Logger').default;
 const { _ } = require('./locale');
 const urlUtils = require('./urlUtils.js');
+const str2ab = require('string-to-arraybuffer');
```

Comment:
I would prefer if we don't add this package as it's unsupported, and I expect not necessary. Node buffer supports many formats - is it not possible to use one of its helper functions to load the content?